linux_os/guide/services/radius/package_freeradius_removed/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: ol7,ol8,rhel6,rhel7,rhel8
+
+title: 'Remove the FreeRadius Server Package'
+
+description: |-
+    The <tt>freeradius</tt> package should be removed if not in use.
+    Is this system a RADIUS server? If not, remove the package.
+    {{{ describe_package_remove(package="freeradius") }}}
+    The freeradius RPM is not installed by default on a {{{ full_name }}}
+    system. It is needed only by the RADIUS servers, not by the
+    clients which use RADIUS for authentication. If the system is not
+    intended for use as a RADIUS Server it should be removed.
+
+rationale: |-
+    Unnecessary packages should not be installed to decrease the attack
+    surface of the system.  While this software is clearly essential on a
+    RADIUS server, it is not necessary on typical desktop or workstation systems.
+
+severity: low
+
```

Comment:
Could you also assign some RHEL CCEs?

Format would be similar to:
`````
identifiers:
    cce@rhel6: xxxx
    cce@rhel7: yyyy
    cce@rhel8: zzzz
    cce@ocp4: aaaa
`````
Where the actual values can be taken from https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt

Just remember to remove the CCE- bit, eg ``cce@rhel6: 82725-3``

Reply:
(and remove and used values from the cce-redhat-avail file)

Reply:
Will push a new version shortly.